### ISSUE TITLE
Presto-cli depends on jline 2.13 in ubuntu 18.04 is faulty

### ISSUE THREAD

**Issue by ideal-hp** [prestodb/presto#10590]
The following error occurs when opening presto-cli:
```
[ERROR] Failed to construct terminal; falling back to unsupported
java.lang.NumberFormatException: For input string: "0x100"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.valueOf(Integer.java:766)
	at jline.internal.InfoCmp.parseInfoCmp(InfoCmp.java:59)
	at jline.UnixTerminal.parseInfoCmp(UnixTerminal.java:233)
	at jline.UnixTerminal.<init>(UnixTerminal.java:64)
	at jline.UnixTerminal.<init>(UnixTerminal.java:49)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at jline.TerminalFactory.getFlavor(TerminalFactory.java:209)
	at jline.TerminalFactory.create(TerminalFactory.java:100)
	at jline.TerminalFactory.get(TerminalFactory.java:184)
	at jline.TerminalFactory.get(TerminalFactory.java:190)
	at jline.console.ConsoleReader.<init>(ConsoleReader.java:240)
	at jline.console.ConsoleReader.<init>(ConsoleReader.java:232)
	at jline.console.ConsoleReader.<init>(ConsoleReader.java:220)
	at com.facebook.presto.cli.LineReader.<init>(LineReader.java:33)
	at com.facebook.presto.cli.Console.runConsole(Console.java:149)
	at com.facebook.presto.cli.Console.run(Console.java:141)
	at com.facebook.presto.cli.Presto.main(Presto.java:32)
```

The direction button will also fail:
presto> ^[[A^[[B^[[D^[[A^[[C^[[D^[[A^[[C

**Comment by ideal-hp** [comment]
I tried to change the jline version to 2.14.6 and the problem will be resolved:
os:  ubuntu-18.04-lts

**depends:**
            <dependency>
                <groupId>jline</groupId>
                <artifactId>jline</artifactId>
                <version>2.14.6</version>
                <exclusions>
                    <exclusion>
                        <groupId>org.fusesource.jansi</groupId>
                        <artifactId>jansi</artifactId>
                    </exclusion>
                </exclusions>
            </dependency>

**Comment by wongjingping** [comment]
I'm getting the exact same set of errors on mac OS X 10.13.5. 
A [PR](https://github.com/prestodb/presto/commit/fa5b91a298cc61a37701e5f35539f9a6bfd112dc) was merged with the stated fix.
Building the latest code for presto-cli (presto-main 0.205) with the dependency fix in [pom.xml](https://github.com/prestodb/presto/blob/master/pom.xml) resolved it.

**Comment by kokosing** [comment]
It is solved in 0.205.

**Comment by ideal-hp** [comment]
good!
